CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+v0.93.0
+------------------------------
+*September 14 2018*
+
+### Changed
+- Added addition focus states to improves accessibility of formToggles.
```

Comment:
_Technically_ this should be under an 'Added' section

Reply:
@xander-marjoram updated 👍